pkg/cmd/cli/cmd/export.go

```diff
@@ -99,7 +101,7 @@ func RunExport(f *clientcmd.Factory, exporter Exporter, in io.Reader, out io.Wri
 
 	mapper, typer := f.Object()
 	b := resource.NewBuilder(mapper, typer, f.ClientMapperForCommand()).
-		NamespaceParam(cmdNamespace).DefaultNamespace().
+		NamespaceParam(cmdNamespace).DefaultNamespace().AllNamespaces(allNamespaces).
 		FilenameParam(explicit, filenames...).
 		SelectorParam(selector).
 		ResourceTypeOrNameArgs(all, args...).
```

Comment:
i'm going to retract my suggestion.  it's probably worth keeping the consistency w/ other resources even though it's a bit weird.